mixer/test/client/rbac_permissive_policy/rbac_permissive_policy_test.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2018 Istio Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package client_test
+
+import (
+	"fmt"
+	"testing"
+
+	"istio.io/istio/mixer/test/client/env"
+)
+
+// Report attributes from a GET request for Rbac permissive mode at policy level.
+const reportAttributes = `
+{
+    "connection.mtls": false,
+    "context.protocol": "http",
+    "context.proxy_error_code": "*",
+    "mesh1.ip": "*",
+    "mesh2.ip": "*",
+    "mesh3.ip": "*",
+    "rbac.permissive.effective_policy_id": "details-reviews-viewer",
+    "rbac.permissive.response_code": "200",
+    "request.headers": {
+        ":method": "GET",
+        ":path": "/echo",
+        ":authority": "*",
+        "x-forwarded-proto": "http",
+        "x-request-id": "*",
+        "user-agent": "Go-http-client/1.1",
+        "content-length": "*",
+        "accept-encoding":"gzip"
+    },
+    "request.host": "*",
+    "request.method": "GET",
+    "request.path": "/echo",
+    "request.scheme": "http",
+    "request.size": 0,
+    "request.time": "*",
+    "request.total_size": 0,
+    "request.url_path": "/echo",
+    "request.useragent": "Go-http-client/1.1",
+    "response.code": 403,
+    "response.duration": "*",
+    "response.headers": {
+        "date": "*",
+        "content-length": "*",
+        ":status": "403",
+        "server": "envoy",
+        "content-type":"text/plain"
+    },
+    "response.size": "*",
```

Comment:
Compared with reportAttributes in #8702,
```
    "response.code": 200,
    "response.headers": {
        ":status": "200",
    },
    "rbac.permissive.response_code": "403"
```
Does global permissive level allow requests by default, and set rbac.permissive.response_code to 403,
while policy permissive level reject requests by default, and set rbac.permissive.response_code to 200?

Reply:
https://github.com/istio/istio/pull/8702 test set permissive mode on globaconfig level, i.e. since rbac is denied by default, so in https://github.com/istio/istio/pull/8702 repsonse code is 200 and permissive resp code is 403.

this PR, response code is 403 since rbac is turn on/denied by default, by because permissive mode is set on policy level, so permissive resp code is 200.

Reply:
Thanks for the explanation.